controller_manager:
  ros__parameters:
    update_rate: 500 # Hz

    effort_controllers:
      type: effort_controllers/JointGroupEffortController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
  
    imu_sensor_broadcaster:
      type: imu_sensor_broadcaster/IMUSensorBroadcaster

effort_controllers:
  ros__parameters:
    joints:
      - joint_l
      - joint_r
    command_interfaces:
      - effort
    state_interfaces:
      - position
      - velocity
      - effort
imu_sensor_broadcaster:
  ros__parameters:
    sensor_name: mobile_imu_sensor
    frame_id: jetson_link